File: python/AKsesoris.py
from Petshop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def get_info(self):
        return super().get_info() + [self.jenis, self.bahan, self.warna]

File: Java/Aksesoris.java
// Child dari PetShop
public class Aksesoris extends PetShop {
    protected String jenis;
    protected String bahan;
    protected String warna;

    public Aksesoris(int id, String namaProduk, int hargaProduk, int stokProduk, 
                     String jenis, String bahan, String warna) {
        super(id, namaProduk, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    @Override
    public void display() {
        super.display();
        System.out.printf(" %-10s | %-10s | %-10s |", jenis, bahan, warna);
    }
}
